tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -0,0 +1,296 @@
+                )
+            creation_time_measures.append(creation_time)
+
+            pv_name = pvc_obj.backed_pv
+            pvc_reclaim_policy = pvc_obj.reclaim_policy
+
+            pod_obj = self.write_file_on_pvc(pvc_obj)
+            pod_obj.delete(wait=True)
+            teardown_factory(pvc_obj)
+            logging.info(f"{msg_prefix} Start deleting PVC number {i + 1}")
+            if pvc_reclaim_policy == constants.RECLAIM_POLICY_DELETE:
+                pvc_obj.delete()
+                pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+                helpers.validate_pv_delete(pvc_obj.backed_pv)
+                deletion_time = helpers.measure_pvc_deletion_time(
+                    self.interface, pv_name
+                )
+                logging.info(
+                    f"{msg_prefix} PVC number {i + 1} was deleted in {deletion_time} seconds."
+                )
+                if deletion_time > accepted_deletion_time:
+                    raise ex.PerformanceException(
+                        f"{msg_prefix} PVC deletion time is {deletion_time} and is greater than "
+                        f"{accepted_deletion_time} seconds."
+                    )
+                deletion_time_measures.append(deletion_time)
+            else:
+                logging.info(
+                    f"Reclaim policy of the PVC {pvc_obj.name} is not Delete;"
+                    f" therefore not measuring deletion time for this PVC."
+                )
+
+        creation_average = self.process_time_measurements(
+            "creation",
+            creation_time_measures,
+            accepted_creation_deviation_percent,
+            msg_prefix,
+        )
+        deletion_average = self.process_time_measurements(
+            "deletion",
+            deletion_time_measures,
+            accepted_deletion_deviation_percent,
+            msg_prefix,
+        )
+
+        # all the results are OK, the test passes, push the results to the codespeed
+        push_to_pvc_time_dashboard(self.interface, "1-pvc-creation", creation_average)
+        push_to_pvc_time_dashboard(self.interface, "1-pvc-deletion", deletion_average)
+
+    def process_time_measurements(
+        self, action_name, time_measures, accepted_deviation_percent, msg_prefix
+    ):
+        """
+           Analyses the given time measured. If the standard deviation of these times is bigger than the
+           provided accepted deviation percent, fails the test
+
+        Args:
+            action_name (str) Name of the action for which these measurements were collected; used for the logging
+            time_measures: A list of time measurements
+            accepted_deviation_percent: Accepted deviation percent, if the standard  deviation of the provided time
+                measurements is bigger than this value, the test fails
+            msg_prefix: A string for comprehensive logging
+
+        Returns
+            The average value of the provided time measurements
```

Comment:
Please add the type of value being returned

Reply:
@keesturam Done.